lib/form/paragraph3.flow

```diff
@@ -143,6 +168,9 @@ renderParagraph(
 	words : [ParaWord] = map(list2array(glued), makeParaWord);
 
 	aw = make(0.0);
+	width = if (widthManager == dummyWigiParagraphWidthManager) make(100000.0) else widthManager.width;
+	next(width, 100000.0);
```

Comment:
define common variable and use in the code.

Reply:
I thought when the manager is defined, we need to use the behaviour from there. Do you suggest to connect it with a segregated behaviour I should create there anyway?

Reply:
sure we should use width from width manager if it is here. I'm talking about the constant 100000.0 you have places in many places in the code